src/ui/hotkey_overlay.rs

```diff
@@ -253,13 +253,13 @@ fn render(
         &Action::SwitchFocusBetweenFloatingAndTiling,
     ]);
 
-    // Screenshot is not as important, can omit if not bound.
-    if binds.iter().any(|bind| bind.action == Action::Screenshot) {
-        actions.push(&Action::Screenshot);
-    }
-
-    // Add actions with a custom hotkey-overlay-title.
     for bind in binds {
+        // Screenshot is not as important, can omit if not bound.
+        if let Action::Screenshot(_) = bind.action {
+            actions.push(&bind.action);
+        }
+
```

Comment:
It is wrong to move this here (you'll get duplicate actions). Change it back, then simply use `matches!(bind.action, Action::Screenshot(_))` as the condition